package.json

```diff
@@ -15,9 +17,10 @@
     "prebuild": "mkdir -p ./dist",
     "build": "harp compile ./src ./dist; cp CNAME dist/; npm run 'build:participants.json'",
     "build:participants.json": "node scripts/concat-json.js ./participants dist/participants.json; cp ./dist/participants.json ./src/",
-    "start": "harp server ./src",
+    "start": "harp server ./src & browser-sync start --proxy 'localhost:9000' --files 'src/**/*.ejs, src/**/*.css', src/**/*.js",
```

Comment:
I would split this into 2 scripts:

* start : harp server...
* dev : yarn run start & browser-sync ...